java/symbolize-shapefile/README.md

```diff
@@ -1,23 +1,31 @@
-# Symbolize a Shapefile
-Override the default renderer of a shapefile when displaying with a FeatureLayer.
+# Symbolize a shapefile
 
-![Symbolize Shapefile App](symbolize-shapefile.png)
+Display a shapefile with custom symbology.
+
+![Image of symbolize shapefile](symbolize-shapefile.png)
+
+## Use case
+
+Feature layers created from shapefiles do not possess any rendering information, and will be assigned with a default symbology. You can apply custom styling to ensure that the content is visible and usable in the context of a specific map. For example, you could use this to visually differentiate between features originating from two different shapefiles, by applying a blue color to one, and a red color to the other.
 
 ## How to use the sample
-Simply run the app.
+
+Pan and zoom around the map to view the symbology renderer set to the feature layer created from the shapefile. 
 
 ## How it works
-1. Create a `ShapefileFeatureTable` by setting the path to a shapefile. 
-1. Create a `FeatureLayer` and set the feature layer table property to the `ShapeFileFeatureTable`.
-1. Create a `SimpleRenderer` to override the default symbology. In this case, a yellow `SimpleFillSymbol` with a red `SimpleLineSymbol` as an outline.
-1. Apply the symbol to the `FeatureLayer` by setting the renderer.
+
+1. Create a `ShapefileFeatureTable`, providing the path to a shapefile.
+2. Create a `FeatureLayer` and associate it with the `ShapeFileFeatureTable`.
```

Comment:
In similar situations the wording is usually something like "Create a `FeatureLayer` using a `ShapeFileFeatureTable`, but this isn't too hard to understand.